README.md

```diff
@@ -131,5 +147,5 @@ role, goal, and reason for each.
 Finally, link to your repo on github in the space below.
 
 Your Answer:
-```text
+```https://github.com/schersh/Project-1
```

Comment:
nice user stories! Will definitely play your game to avoid dementia :)

Consider using [markdown syntax](https://help.github.com/articles/markdown-basics/) to format your readme.